engines/query-sparql/test/QuerySparql-test.ts

```diff
@@ -9,10 +9,11 @@ import type { QueryBindings, QueryStringContext } from '@comunica/types';
 import type * as RDF from '@rdfjs/types';
 import 'jest-rdf';
 import arrayifyStream from 'arrayify-stream';
+import * as n3 from 'n3';
 import { Store } from 'n3';
 import { DataFactory } from 'rdf-data-factory';
 import { Factory } from 'sparqlalgebrajs';
-import { QueryEngine } from '../lib/QueryEngine';
+import { QueryEngine } from '../lib';
```

Comment:
This _may_ be the cause of the coverage drop.

Reply:
seems to fix it!